src/modes/direct_select.js

```diff
@@ -73,6 +79,21 @@ module.exports = function(ctx, opts) {
     selectedCoordPaths = [about.coord_path];
   };
 
+  function pathsToCoordinates(paths) {
+    return paths.map(coord_path => feature.getCoordinate(coord_path));
+  }
+
+  function coordinatesToPoints(coordinates) {
+    return coordinates.map(coords => ({
+      type: Constants.geojsonTypes.FEATURE,
+      properties: {},
```

Comment:
As said on the comment I left in `store.js` I think converting this data into features should be done in `api.js`. This is to keep with how `store.js` currently works (it doesn't store user presented GeoJSON) and to keep the Draw API interface immutable.